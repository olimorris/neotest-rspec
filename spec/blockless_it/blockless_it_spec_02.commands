:2
:lua require("neotest").run.run()
