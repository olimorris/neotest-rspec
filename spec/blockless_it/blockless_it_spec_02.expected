./SPEC/BLOCKLESS_IT/BLOCKLESS_IT_SPEC.RB@@PASSED@@this works and gets tested by neotest
